components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint.util/pom.xml

```diff
@@ -64,6 +64,11 @@
         </dependency>
         <dependency>
             <groupId>org.wso2.carbon.identity</groupId>
+            <artifactId>org.wso2.carbon.identity.application.authentication.framework</artifactId>
```

Comment:
It would be better if we can add getServicesUrl functionality to identity core util, so we can avoid adding this new dependency, and also get ServicesUrl would be possible to use in any identity component, WDYT?